config/config.go

```diff
@@ -10,6 +10,8 @@ import (
 type Config struct {
 	Path                  string       `json:"path" yaml:"path" mapstructure:"path" `
 	AppID                 string       `json:"appId" yaml:"appId" mapstructure:"appId"`
+	ReRecord              string       `json:"rerecord" yaml:"rerecord" mapstructure:"rerecord"`
```

Comment:
Why have you added the `ReRecord` flag in the global config struct? Please remove it from here.